README.md

```diff
@@ -1,24 +1,58 @@
-# README
+# Recipe-app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- ROR
+- Postgres
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get a local copy up and running follow these simple example steps.
 
-* Database creation
+### Prerequisites
 
-* Database initialization
+You must have Ruby installed in your computer.
 
-* How to run the test suite
+### Clone this repo
 
-* Services (job queues, cache servers, search engines, etc.)
+`https://github.com/Tadesse-Alemayehu/Recipe-app`
 
```

Comment:
- [ **OPTIONAL** ] I think it will be better if you update this code line `https://github.com/Tadesse-Alemayehu/Recipe-app` to this: `git clone https://github.com/Tadesse-Alemayehu/Recipe-app` in order to have the entire command on how to clone the repo.

Reply:
It is done!

Reply:
It is done!